router/xgress_edge/accept.go

```diff
@@ -45,6 +46,7 @@ func (handler edgeBindHandler) BindChannel(ch channel2.Channel) error {
 		listener:     handler.listener,
 		fingerprints: fpg.FromCerts(ch.Certificates()),
 		ch:           ch,
+		idSeq:        math.MaxUint32 / 2,
```

Comment:
For curiosity reasons, why max int / 2?

Reply:
The id ranges are split into two. 0 to (maxuint32 / 2 - 1) for client generated ids and maxuint32 / 2 -> maxuint32 for server generated ids. This way we don't run into id conflicts.
Eugene pointed out that http2 does something similar, but they use even number for one and odd numbers for the other.